.github/workflows/tests.yml

```diff
@@ -72,9 +86,9 @@ jobs:
         run: |
           choco install -y -r --no-progress pandoc
 
-      - name: "Run tox for ${{ matrix.python-version }}"
+      - name: "Run tox"
         run: |
-          python -m tox
+          ${{ env.venv-path }}/python -m tox -m ci-tests
```

Comment:
The Tox invocation changes from a bare run of Tox (which was intercepted by tox-gh and transformed to a single run of the default test environment) to use Tox 4's new labels. This allows the `tox.ini` file to control which environments the `ci-tests` label is associated with.